docs/virtual_machines/presets.md

```diff
@@ -55,8 +57,8 @@ applied to a `VirtualMachineInstance`, the `VirtualMachineInstance` will
 be marked with an Annotation upon completion.
 
 Any domain structure can be listed in the `spec` of a
-`VirtualMachineInstancePreset`, e.g. Clock, Features, Memory, CPU, or
-Devices such as network interfaces. All elements of the `spec` section
+`VirtualMachineInstancePreset` (Clock, Features, Memory, CPU, or
```

Comment:
I don't think this list is all encompasing. To me the parenthesis implies a greater exactness to the list of examples--but I don't feel very strongly on this point if we prefer it this way.

Reply:
Reverting the txt passes non-ascii check. I'll just do that and keep the diff minimal